compiler/src/test/resources/bugs/apps_1197_native_frag_default_unit.wdl

```diff
@@ -0,0 +1,36 @@
+version 1.0
+
+workflow apps_1197_native_frag_default_unit {
+
+    # Memory
+    call apps_1197_default_isntance {}
+
+    # Conditional mem
+    Boolean a = true
+    if (a) {
+        call apps_1197_default_isntance as conditional_mem {}
+    }
+
+
+}
+
+
+task apps_1197_default_isntance {
```

Comment:
```suggestion
task apps_1197_default_instance {
```

Reply:
fixed in [868e433](https://github.com/dnanexus/dxCompiler/pull/344/commits/868e43320edaacb2f5cedbc686437bdb6ae8f666)